Update to FAQ Change Domain Added (#1118)
btcpayserver/btcpayserver-doc: docs/FAQ/Deployment.md

Notice: If your BTCPay Server install has more than one domain (for example `WOOCOMMERCE_HOST` or `BTCPAY_ADDITIONAL_HOSTS`) you will need to modify your config for each domain name. The example above only covers 1 domain name called `btcpay.domain.com`.

## How to change your BTCPay Server domain name ? 

### Setting up DNS Records

You bought a domain and now want to connect that to your BTCPay Server. 

> The hosting party's usually has a page to manage your domain. 

Reviewer: ```suggestion
The hosting party usually has a page to manage your domain. 
```

Here you will find the `DNS records` page and add a `CNAME` record. 

> In this record you will make sure, it points to the provided Domain by your VPS hoster.

Reviewer: ```suggestion
In this record you will make sure, it points to the provided domain by your VPS hoster.
```

You could also do this by IP address, but then instead of a `CNAME record` it would be an `A Record`.

This is an example of how this would look at [gandi.net](https://gandi.net/)

![Gandi3](../img/Gandi3.png "Gandi add DNS record")

### Change domain name in BTCPay Server settings

> In BTCPay server you go to the `Server Settings` menu, and then into the tab `Maintenance`.

Reviewer: ```suggestion
In BTCPay Server you go to the `Server Settings` menu, and then into the tab `Maintenance`.
```

> Here you will find a field to replace your old by the new set domain, it might take a few seconds te work. 

Reviewer: ```suggestion
Here you will find a field to replace your old by the new set domain, it might take a few seconds to update. 
```

![Maintenance domain name](../img/changedomain.png "BTCPay Server Change Domain name")

Now enter the new set domain in the address bar and see if it works! 

![Maintenance2](../img/Maintenance2.png "BTCPay Server Change Domain name")

### Change domain on command line

Connect to your server through SSH.
 
Example :

```bash
ssh btcpayserver@myawesomedemobtcpay.westeurope.cloudapp.azure.com
```

> Enter your `password` set for azure, and change the domain name.

Reviewer: ```suggestion
Enter your password and change the domain name.
```
